doc/rdoc/markup_reference.rb

```diff
@@ -370,30 +399,30 @@
 #   # ============Still a Heading (Level 6)
 #   # \== Not a Heading
 #
+# A heading may contain only one type of nested block:
+#
+# - {Text Markup}[rdoc-ref:RDoc:MarkupReference@Text+Markup]:
+#   italic, bold, monofont.
```

Comment:
Also ditto here, we can remove the types of text markup.

Reply:
Done.